lib/modules/web3/transports.ts

```diff
@@ -12,7 +12,8 @@ export function getTransports(chain: Chain) {
   const gqlChain = getGqlChain(chain.id as SupportedChainId)
   const overrideRpcUrl = rpcOverrides[gqlChain]
   const fallbackRpcUrl = rpcOverrides[gqlChain]
-  return fallback([http(overrideRpcUrl), http(), http(fallbackRpcUrl)])
+  if (overrideRpcUrl) return fallback([http(overrideRpcUrl), http(fallbackRpcUrl), http()])
+  return fallback([http(), http(fallbackRpcUrl)])
```

Comment:
Shouldn't this last return be this then?
```suggestion
  return fallback([http(fallbackRpcUrl), http()])
```

Reply:
If overrideRpcUrl is defined then the priority order is:
1. overrideRpcUrl
2. fallbackRpcUrl
3. default (http)

But when overrideRpcUrl is not defined we prefer the default over the fallback one so the priority order is: 
1. defautl (http)
2. fallbackRpcUrl